### ISSUE TITLE
Back End code?

### ISSUE THREAD

**Issue by jamesmbowler** [GoThinkster/redux-review#20]
I saw in the readme there was an offer to release backend code for this demo .. Is that possible?

Thanks, again, this is great.

**Comment by EricSimons** [comment]
Yup! It's coming in 2 weeks -- I'll post a link here once it's ready. Thanks for the kind words btw :)

**Comment by jamesmbowler** [comment]
Is it ready yet?

**Comment by EricSimons** [comment]
It's not quite ready but here's the latest version of the Node backend (everything _should_ work): https://github.com/gothinkster/productionready-node-api

Would love your feedback & will post back here once it's 100% done.

**Comment by EricSimons** [comment]
Sorry about the optimism around our deadlines -- we've been swamped the past few months but are finally coming up for air.

**Comment by jamesmbowler** [comment]
No worries - this looks great so far, thanks.
